src/components/Dialog/Dialog.tsx

```diff
@@ -65,7 +65,6 @@ export class Dialog extends React.Component<IDialogProps, any> {
             ariaDescribedBy={ subText ? id + '-subText' : '' }
             onDismiss={ onDismiss }
             >
-            <div className={ dialogClassName }>
```

Comment:
I'm confused; the issue was setting maxwidth:

$Dialog-default-max-width: 340px;